DEVDOCS-78 - Installation guide update

nikolalardev opened the pull request:
Update of the installation guide

- remove env.php file section
- adding bin/magento install command information
- remove web wizard section
- update Magento with Mage-OS

rhoerr commented:
Thanks! I reviewed and made some more changes.

Could you test to make sure all required options are covered in the example command? Specifically I think there might be elasticsearch params not covered.

nikolalardev commented:
Hello @rhoerr ,

I tested the run of the command `bin/magento setup:install` without any parameters and it is working, it doesn't require any parameters. However, I added the configuration for the search engine OpenSearch. I moved the configuration for `base_url , web/secure/use_in_frontend, web/secure/use_in_adminhtml, web/seo/use_rewrites` to additional post-install commands because were marked as deprecated.
--base-url-secure=BASE-URL-SECURE                                                          Base URL for SSL connection. **Deprecated**, use config:set with path web/secure/base_url
